health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -0,0 +1,701 @@
+      const lowestHierarchyCodes = sessionData?.["HCM_CAMPAIGN_SELECTING_BOUNDARY_DATA"]?.boundaryType?.selectedData
+        ?.filter((i) => i.type === lowestHierarchy)
+        ?.map((j) => j.code);
+      setAllLowestHierarchyCodes(lowestHierarchyCodes);
+    }
+  }, [lowestHierarchy]);
+
+  const getFileStoreId = () => {
+    switch (formData?.name) {
+      case "uploadFacilityMapping":
+        return sessionData?.["HCM_CAMPAIGN_UPLOAD_FACILITY_DATA"]?.uploadFacility?.uploadedFile?.[0]?.filestoreId;
+        break;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)